metadata/network.go

```diff
@@ -66,6 +67,7 @@ var MainnetDefinition = NetworkDefinition{
 	AccessPolicyOracleAddress: "https://trust.mysterium.network/api/v1/access-policies/",
 	BrokerAddresses:           []string{"nats://broker.mysterium.network"},
 	TransactorAddress:         "https://transactor.mysterium.network/api/v1",
+	AffiliatorAddress:         "https://affiliator.mysterium.network/api/v1",
```

Comment:
not sure about it but I think it would be nice to use just "https://affiliator.mysterium.network" in order to be able to call both v1 and v2 endpoints in case we have some in the future, maybe thats a bad choice for some reason